CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to the "vscode-camelk" extension will be documented in this
 
 ## 0.0.32
 
+- Added extensions as dependencies
```

Comment:
From end-user point of view, it might interesting to mention something like `these 2 extensions are now installed automatically when installing VS Code Camel K`

Reply:
You are totally right